client/packages/invoices/src/InboundShipment/DetailView/ContentArea/columns.ts

```diff
@@ -233,35 +239,27 @@ export const useInboundShipmentColumns = () => {
         },
       ],
       {
-        label: 'label.sell',
-        key: 'sellPricePerPack',
+        label: 'label.cost-per-unit',
+        key: 'costPricePerUnit',
```

Comment:
Changed this to cost per unit to match with OS sell price per unit... The pack sizes are can be all different, so not sure how useful pack per unit price really is?